lib/automation.rb

```diff
@@ -8,7 +8,16 @@ def self.available_models
         FROM llm_models
       SQL
 
-      values.each { |value_h| value_h["id"] = "custom:#{value_h["id"]}" }
+      values =
+        values
+          .filter do |value_h|
+            value_h["id"] > 0 ||
+              SiteSetting
+                .ai_automation_allowed_seeded_models
```

Comment:
```suggestion
                .ai_automation_allowed_seeded_models_map
```

I believe you can drop the `.split("|")` and use this helper method instead.